libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -335,6 +344,15 @@ bool AP_Arming::battery_checks(bool report)
         }
     }
 
+    if (((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_BATTERY)) && 
+        (_min_voltage > 0.0f)) {
+        if (_battery.voltage() < _min_voltage) {
```

Comment:
I think so, it's off by default. I can see times where both would want to be checked before launch